### PR TITLE
Update nginx.md

### PR DESCRIPTION
**Pull request by PiotrBracha**
Few improvements inside the Nginx's configuration.

| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | I use Prestashop 1.7.8.8 with Nginx 1.22.1
| Description?  | My PR includes adding more security headers, some improvements in access/error log files and regarding the Nginx docs - the proper use of try_files instead of "if" condition
| Fixed ticket? | it does not fix any issue

### CODE REVIEW

**Review by tleon — APPROVED** [review]

**Review by matks — COMMENTED** [review]
I'm still a newbie in NGINX 😆 

Would love a review from people better skilled - @jokesterfr ? @JMorinPS ?

**Comment by matks** [comment]
One more try @jokesterfr @JMorinPS @djodjo3 😄

**Comment by kpodemski** [comment]
cc @jokesterfr

File: basics/installation/advanced/nginx.md [hunk 1]
```diff
@@ -26,8 +26,8 @@ server {
     server_name example.com www.example.com;
 
     # [EDIT] Path to your domain Nginx logs.
-    access_log /var/log/nginx/example.com-access.log;
-    error_log /var/log/nginx/example.com-error.log;
+    access_log /var/log/nginx/example.com-access.log combined;
```
Comment (kpodemski) [inline basics/installation/advanced/nginx.md:30]:
```suggestion
    # more details: https://docs.nginx.com/nginx/admin-guide/monitoring/logging/#setting-up-the-access-log
    access_log /var/log/nginx/example.com-access.log combined;
```

File: basics/installation/advanced/nginx.md [hunk 2]
```diff
@@ -26,8 +26,8 @@ server {
...
+    error_log /var/log/nginx/example.com-error.log info;
```
Comment (kpodemski) [inline basics/installation/advanced/nginx.md:32]:
```suggestion
    # more details: https://nginx.org/en/docs/ngx_core_module.html?#error_log
    error_log /var/log/nginx/example.com-error.log info;
```

**Review by kpodemski — APPROVED** [review]
Eventually, I did the research, and all the changes are valid. Thanks @PiotrBracha

**Pull request merged**